README.md

```diff
@@ -63,16 +63,37 @@ x-kusk:
 ## Homebrew
 `brew install kubeshop/kusk/kgw`
 
-## Latest release on Github
-`go install github.com/kubeshop/kusk-gateway-cli@$VERSION`
+## Go install the latest release on Github
+`go install github.com/kubeshop/kgw@latest`
+
+To install a particular version: replace `latest` with the version number
+
+You can get a list of the available kgw versions from our [releases page](https://github.com/kubeshop/kgw/releases)
+
+## Easy install script
+This will install `kgw` into `/usr/local/bin/kgw`
+
+```sh
+bash < <(curl -sSLf https://github.com/kubeshop/kgw/blob/main/scripts/install.sh)
```

Comment:
I think Testkube guys created the nice script, but this works only on MacOS, while fails on Linux when trying to move the binary to /usr/local/bin since root rights are needed. We'll need to add a note to run this with root rights for Linux users.

Reply:
yeah and macos folks would use brew anyway